qemu/tests/block_hotplug.py

```diff
@@ -64,6 +68,40 @@ def run_sub_test(params, plug_tag):
                 return True
         return None
 
+    def get_disk_size(did):
+        """
+        Get the disk size from guest.
+
+        :param did: the disk of id, e.g. sdb,sda for linux, 1, 2 for windows
+        :return: the disk size
+        """
+        if params['os_type'] == 'linux':
+            size = utils_disk.get_linux_disks(session)[did][1].strip()
+        else:
+            random_str = random.sample(string.ascii_letters + string.digits, 4)
```

Comment:
It seems vt has a wrapper for  random_str, please have a look, but this is fine for me

Reply:
Thanks, will update it.